Gulpfile.coffee

```diff
@@ -190,5 +212,34 @@ gulp.task 'serve', ['watch'], ->
                 res.setHeader('Content-Type','image/svg+xml');
             next()
     ]
-
   connect.server(config)
+
+gulp.task '_tdd', (done) ->
+  buildAndTest(true, done)
+  return # Since this is async
+
+
+# -----------------------------------------------------------------------
+#  Public Tasks
```

Comment:
:+1:  I like the composition of the public tasks  :smile:  would it make sense to even pull the above private methods into a separate file, and require that in this main Gulpfile where the publics will sit?